flink-optimizer/src/test/java/org/apache/flink/optimizer/DistinctCompilationTest.java

```diff
@@ -71,8 +71,8 @@ public void testDistinctPlain() {
 			assertEquals(reduceNode, sinkNode.getInput().getSource());
 
 			// check that both reduce and combiner have the same strategy
-			assertEquals(DriverStrategy.SORTED_GROUP_REDUCE, reduceNode.getDriverStrategy());
-			assertEquals(DriverStrategy.SORTED_GROUP_COMBINE, combineNode.getDriverStrategy());
+			assertEquals(DriverStrategy.SORTED_REDUCE, reduceNode.getDriverStrategy());
+			assertEquals(DriverStrategy.SORTED_PARTIAL_REDUCE, combineNode.getDriverStrategy());
```

Comment:
Add a test that sets a `CombineHint.HASH` and check that the `combineNode` driver strategy is `HASHED_PARTIAL_REDUCE`?